### ISSUE TITLE
Automation: Big Neon : Test 24: Box Office: Refund Tickets

### ISSUE THREAD

**Issue by Tas4tari** [big-neon/bn-web#1789]
**Pre-conditions:**

- User must be logged in as Org admin

**Steps:**

1. Click on "Events" on the left side bar
2. Look for the event you would like to search for the sold tickets
3. Click on the 3 dots in the top right corner of the event
4. Click on "Dashboard"
5. Click on "Tools"
6. Click on "Manage Orders"
7. Search for the guest (First name, Last name, First and last name, email address or contact number)
8. Guest was found successfully
9. Select the tickets you would like to refund (Tick the box on the left of the order)
10. Check if the "Total value" amount is correct
11. Click on "Refund" in the bottom right corner
12. Received a pop up screen with "Refund"
13. Click on "Refund $..."
14. Refund successful pop up screen should be received
15. Click on "Close"
16. Refund should be done successfully

**Comment by mihajlokolesar** [comment]
Hello @Tas4tari ,
In step 5 there is click on Tools and step 6 on Manage Orders. Since there is no Manage Orders in tools tab(f, (as mentioned in task  #1788 comments). Can I assume that for step 5 it should be click on Orders tab and then click on Manage Orders from drop down menu?
Also refund popup screen is a bit different. there is a possibility to state reason for cancellation, and select additional tickets from orders for refund. Any preference for these steps?

**Comment by Tas4tari** [comment]
@mihajlokolesar Yes please, click on "orders", so the designs may have changed, I will confirm and revert
